pkg/gctuner/finalizer_test.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build enable_flaky_tests
```

Comment:
`cgroup_cpu_test.go` also need

Reply:
The test failed in `cgroup_cpu_test.go` we discussed before may only occur in some OS env. I don't know whether it is appropriate to classify it into flaky tests.

Reply:
Done